### DISCUSSION TITLE
Horizontal rule fills entire width

### DISCUSSION THREAD

**Discussion by stefanv** [minad/org-modern#174] [Q&A] [answered]
The org-modern horizontal rule fills the entire display, such that when moving the cursor over the line the whole buffer "disappears" as the cursor jumps to the end of the line.

Could we make % of width adjustable?

Cursor on regular line:

![2024-02-22_10:57:16](https://github.com/minad/org-modern/assets/45071/5ac94faa-f426-4a84-9115-7a1b541e4cca)

Cursor on horizontal rule:

![2024-02-22_10:57:25](https://github.com/minad/org-modern/assets/45071/bd0ac5ef-bd70-413e-8742-acb72f30c06e)

**Comment by minad** [comment]
Hi! I don't see the issue with the cursor. Do you have a recipe based on emacs -Q such that I can reproduce the problem? You can also set org-horizontal-rule to a replacement string as a workaround, e.g., `"____________________"`.

**Reply by stefanv** [reply, marked as answer]
Thanks for responding @minad, and sorry, I should have isolated the configuration in my initial report.

It looks like some weird interaction between disabling fringe mode and my usage of `org-modern-indent`:

```lisp
; Disable fringe until we find a use for it
(use-package fringe
  :config
  (set-fringe-mode 0))

;; Packaging
(require 'package)
(package-initialize)

;; https://github.com/jdtsmith/org-modern-indent
(use-package org-modern-indent
  :vc (:fetcher github :repo jdtsmith/org-modern-indent)
  :config
  ;; ;; work around org-modern-indent not supporting pre-headline text, from Charl Botha
  ;; https://github.com/jdtsmith/org-modern-indent/issues/10#issuecomment-1671726529
  (add-hook 'org-mode-hook (lambda() (aset org-indent--text-line-prefixes 0 (propertize " " 'face 'org-indent)))))

(use-package org
;;  :after (org-variable-pitch)
  :mode ("\\.org\\'" . org-mode)
  :config
  (setq org-startup-indented t))

(use-package org-modern
  :hook
  (org-mode . org-modern-mode))
```

Given that I pad each line by one space, it's not `org-modern`'s fault; sorry for the noise.

For now, I'm going with:

```lisp
(setq org-modern-horizontal-rule "_________________________________________________________________________________________________")
```
